statsmodels/multivariate/factor.py

```diff
@@ -372,3 +527,55 @@ def plot_loadings(self, loading_pairs=None, plot_prerotated=False):
         return plot_loadings(loadings, loading_pairs=loading_pairs,
                              title=title, row_names=self.endog_names,
                              percent_variance=var_explained)
+
+    """
+    Returns the fitted covariance matrix.
+    """
+    @cache_readonly
+    def fitted_cov(self):
+        c = np.dot(self.loadings, self.loadings.T)
+        c.flat[::c.shape[0]+1] += self.uniqueness
+        return c
+
+    """
+    The standard errors of the uniquenesses.
+
+    If excess kurtosis is known, provide as `kurt`.  Standard errors
+    are only available if the model was fit using maximum likelihood.
+    If `endog` is not provided, `nobs`must be provided to obtain
+    standard errors.
+    """
+    @cache_readonly
+    def uniq_stderr(self, kurt=0):
```

Comment:
Fixed; but I don't think the standard errors require elliptical distributions.  There are some regularity conditions A-D in the Bai and Li paper.

Reply:
You are right, they don't have anything like an elliptical assumption.
My guess is that it is because this looks at one sigma_j at a time (and it is the Sigma is diag) and  including kurt allows for arbitrary 4th moment. (Just staring at their proof for a few minutes doesn't help.)

In cov_cov and cov_corr (covariance of sampling covariance matrix) the full matrix of 4th moments is needed in the non-elliptical case, and simplifies to something similar in the elliptical and normal cases.